settings.gradle.kts

```diff
@@ -52,6 +53,10 @@ include(":samples:tlssurvey")
 include(":samples:unixdomainsockets")
 include(":container-tests")
 
+if (mavenBuild.toBoolean()) {
+  include(":maven-tests")
```

Comment:
I'm not sure why we want to build these sources from Gradle and Maven. As I said above, would prefer to always build with Maven which should eliminate the need to ever build with Gradle.

Reply:
My preference is for CI to always build with Maven, and locally I don’t have to think about it?